Allow to use the "Show more" and "Show less" buttons with the keyboard.

diff --git a/view/frontend/web/js/attribute-filter.js b/view/frontend/web/js/attribute-filter.js
index e6d7d73..2505ee9 100644
--- a/src/module-elasticsuite-catalog/view/frontend/web/js/attribute-filter.js
+++ b/src/module-elasticsuite-catalog/view/frontend/web/js/attribute-filter.js
@@ -153,7 +153,10 @@ define([
         /**
          * Callback for the "Show more" button
          */
-        onShowMore: function () {
+        // BEGIN PATCH
+        onShowMore: function (uiClass, event) {
+            typeof event?.preventDefault === 'function' && event.preventDefault();
+            // END PATCH
             if (this.hasMoreItems) {
                 this.loadAdditionalItems(this.onShowMore.bind(this));
             } else {
@@ -175,7 +178,10 @@ define([
         /**
          * Callback for the "Show less" button
          */
-        onShowLess: function () {
+        // BEGIN PATCH
+        onShowLess: function (uiClass, event) {
+            typeof event?.preventDefault === 'function' && event.preventDefault();
+            // END PATCH
             this.expanded(false);
         },

diff --git a/view/frontend/web/template/attribute-filter.html b/view/frontend/web/template/attribute-filter.html
index a4a111b..8d17273 100644
--- a/src/module-elasticsuite-catalog/view/frontend/web/template/attribute-filter.html
+++ b/src/module-elasticsuite-catalog/view/frontend/web/template/attribute-filter.html
@@ -33,7 +33,9 @@

 <div class="actions" data-bind="visible: enableExpansion()">
     <div class="secondary">
-        <a class="action show-more" data-bind="visible: displayShowMore(), event: {click: onShowMore}"><span data-bind="text: showMoreLabel"></span></a>
-        <a class="action show-less" data-bind="visible: displayShowLess(), event: {click: onShowLess}"><span data-bind="text: showLessLabel"></span></a>
+        <!-- BEGIN PATCH -->
+        <a class="action show-more" href="#" data-bind="visible: displayShowMore(), event: {click: onShowMore}"><span data-bind="text: showMoreLabel"></span></a>
+        <a class="action show-less" href="#" data-bind="visible: displayShowLess(), event: {click: onShowLess}"><span data-bind="text: showLessLabel"></span></a>
+        <!-- END PATCH -->
     </div>
 </div>
